tests/debugger/expected_line_probes.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
For that simple data structure, you can put it directly in the python source. And lucky you, it's the same syntax :)